News/Sceen/Main/ViewControllers/MainViewController.swift

```diff
@@ -0,0 +1,142 @@
+//
+//  MainViewController.swift
+//  News
+//
+//  Created by Mohamed Elboraey on 13/05/2025.
+//
+
+import UIKit
+import Combine
+
+class MainViewController: UIViewController {
+//    MARK: - Proberities
```

Comment:
```suggestion
    // MARK: - Properties
```
I would suggest enabling code spelling check in the Xcode to avoid such typos.